crates/compiler/gen_dev/src/lib.rs

```diff
@@ -1250,9 +1276,7 @@ trait Backend<'a> {
                 self.load_literal_symbols(args);
                 self.build_fn_call(sym, fn_name, args, arg_layouts, ret_layout)
             }
-            other => {
-                eprintln!("maybe {other:?} should have a custom implementation?");
-
+            _other => {
```

Comment:
My this better because it is simpler?

The reason for this error was to cause us to explicitly list out function above here. By being explicit, I believe we can eventually error out on functions that won't exist. Though maybe that is a wrong/silly assumption. Any valid mono ir should only contain functions that we can call and nothing else. I guess being explicit is minorly faster, but probably not any in way that matters to execution time. Maybe in the long term we should remove all the special functions above like `Symbol::STR_IS_VALID_SCALAR` and just default all unknown functions through this branch including all builtins. Thoughts?

Reply:
the mechanism no longer works. We have lots of symbols in the standard library now that are not top-level functions.

also yes mono should be valid. and yes we can/should clean this up now because that function does not do a lot of useful work.